doc/release/upcoming_changes/27735.new_feature.rst

```diff
@@ -0,0 +1,2 @@
+* UFuncs new support `__dict__` attribute and allow overriding
+  `__doc__` (either directly or via `ufunc.__dict__["__doc__"]`).
```

Comment:
Please also point out `__module__`, and `__qualname__`.

Reply:
Done!